linux_sql/scripts/host_info.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+#ARGUMENTS
+psql_host=$1
+psql_port=$2
+db_name=$3
+psql_user=$4
+psql_password=$5
+
+#Argument Validation
+if [ "$#" -ne 5 ]; then
+  echo "ERROR: Illegal number of parameters"
+  echo "  Script Usage: host_info.sh psql_host psql_port db_name psql_user psql_password"
+  exit 1
+fi
+
+#MAIN
+#Obtain host hardware specifications
+lscpu_out=`lscpu`
+proc_mem_out=`cat /proc/meminfo`
+vmstat_out=`vmstat -t`
+
+#Parse host hardware specifications
+hostname=$(hostname -f)
+cpu_number=$(echo "$lscpu_out" | egrep "^CPU\(s\):" | awk '{print $2}' | xargs)
```

Comment:
create a new function for `$(echo "$lscpu_out" | egrep "${pattern}" ` (optional)